media/starboard/sbplayer_bridge.cc

```diff
@@ -1143,9 +1143,7 @@ void SbPlayerBridge::OnPlayerStatus(SbPlayer player,
     if (sb_player_state_initialized_time_.is_null()) {
       sb_player_state_initialized_time_ = Time::Now();
     }
-    sbplayer_interface_->Seek(player_, preroll_timestamp_, ticket_);
-    SetVolume(volume_);
```

Comment:
nit: as `SetVolume` and `SetPlaybackRate` handled by PipelineImpl, we should add a description in the commit message to mention why they are removed.

Reply:
Done